packages/remirror__extension-list/src/list-item-shared-extension.ts

```diff
@@ -0,0 +1,19 @@
+import { KeyBindings, PlainExtension } from '@remirror/core';
+
+import { sharedLiftListItem, sharedSinkListItem } from './list-commands';
+
+/**
+ * Provides some shared keymaps used by both `listItem` and `taskListItem`
+ */
+export class ListItemSharedExtension extends PlainExtension {
+  get name() {
+    return 'listItemSharedCommands' as const;
+  }
+
+  createKeymap(): KeyBindings {
+    return {
+      Tab: sharedSinkListItem(this.store.extensions),
+      'Shift-Tab': sharedLiftListItem(this.store.extensions),
+    };
+  }
+}
```

Comment:
```suggestion
}

declare global {
  namespace Remirror {
    interface AllExtensions {
      listItemShared: ListItemSharedExtension;
    }
  }
}
```

Add the extension to the global type.